src/navigator/IFrameNavigator.ts

```diff
@@ -159,15 +160,16 @@ export interface ReaderConfig {
   bookmarks?: BookmarkModuleConfig;
   highlighter?: TextHighlighterConfig;
   injectables: Array<Injectable>;
+  injectablesFixed: Array<Injectable>;
   useLocalStorage?: boolean;
   attributes?: IFrameAttributes;
   services?: PublicationServices;
 }
 
 /** Class that shows webpub resources in an iframe, with navigation controls outside the iframe. */
 export default class IFrameNavigator implements Navigator {
-  iframe: HTMLIFrameElement;
-  iframe2: HTMLIFrameElement;
+  iframes: Array<HTMLIFrameElement> = [];
```

Comment:
Is this always supposed to have exactly two iframes in it? If so it might make sense to type it as a tuple: 
```ts
iframes: [HTMLIFrameElement, HTMLIFrameElement];
```
that way Typescript know that there are always exactly two items in it, and will keep us from trying to access `iframes[2]` or something, and we can also be sure that `iframes[0]` and `iframes[1]` will always be defined.

Reply:
at some point there might be 3 iframes, there are fixed layout epubs that have 3, but for now we will either have 1 or two, in fixed layout you can have spreads of 1 to 3 i believe, but we are just using 1 or 2 here

Reply:
i am using the user-settings column count as page count here, where the user could set the book to show in one page vs two, or vise versa. some fixed layout books are not laid out properly and have two pages in one spread, so it's good to be able to only show one in that case.